content/scripts/check-links.sh

```diff
@@ -9,6 +9,26 @@ if [ -n "$REJECT_PATH" ]; then
 	REJECT_REGEX_ARG="--reject-regex ${URL}${REJECT_PATH}"
 fi
 
+redirects_file="$(mktemp)"
+cat content/redirects.txt| awk -F' ' '{print $1}' | grep -v '#' | sort | uniq > "$redirects_file"
```

Comment:
This should probably be a more specific grep to only strip lines that start with `#`...